types/bcrypt/index.d.ts

```diff
@@ -121,7 +121,7 @@ export declare function hash(
     data: string | Buffer,
     saltOrRounds: string | number,
     callback: (err: Error | undefined, encrypted: string) => any,
-): void;
+): Promise<string>;
```

Comment:
```suggestion
): void;
export declare function hash(
    data: string | Buffer,
    saltOrRounds: string | number,
): Promise<string>;
```

Reply:
Oops, you’re right—I didn’t see that at the top. The return for this one should probably just be changed to `void`, then:

```suggestion
): void;
```